### DISCUSSION TITLE
Server on specific url

### DISCUSSION THREAD

**Discussion by Klorestz** [bluenviron/mediamtx#3055] [Questions]
### Question

Hello, can I only run the server on a specific url? For example, only on rtmps://localhost:1935/mystream, and not use the other url?

**Comment by aler9** [comment]
Hello, open the configuration file and delete the `all_others` path from the `paths` section. Then add just `mystream`:

```yml
paths:
   mystream: 
```

In this way it will be possible to use the server only to publish to mystream and read to mystream.

If you're also interested into limiting available protocols, disable everything except RTMP:

```yml
rtsp: no
srt: no
webrtc: no
hls: no
```

[a comment by an automated account is not shown]
